src/main/java/com/jfrog/ide/common/nodes/subentities/ScanType.java

```diff
@@ -0,0 +1,21 @@
+package com.jfrog.ide.common.nodes.subentities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ScanType {
```

Comment:
This name is very general, and this enum doesn't include all types of scans (like SCA).
Suggestions: AdvancedScanType or AdvancedSecurityScanType or JasScanType.